scripts/schema_bump_dry_run_genes/gene_bump_dry_run.py

```diff
@@ -201,11 +201,10 @@ def generate_deprecated_public(base_url: str, diff_map: Dict) -> Dict:  # type:
             public_deprecated, is_deprecated_genes_found = compare_genes(dataset, diff_map, public_deprecated)
             if is_deprecated_genes_found:
                 run_reporter.public_deprecated_datasets += 1
-        except Exception as e:
+        except Exception:
             run_reporter.public_errored_datasets += 1
             logger.error(
                 f"Error processing public dataset {dataset['dataset_id']}, in collection {dataset['collection_id']}: "
-                f"{e}"
```

Comment:
is there any reason why we would not want to log the actual exception that occurred?